src/plugins/sessionPlayer/plugin.js

```diff
@@ -276,251 +274,230 @@ define(['playbackManager', 'events', 'serverNotifications', 'connectionManager']
         sendCommandByName(this, 'VolumeUp');
-    };
+    }
 
-    SessionPlayer.prototype.volumeDown = function () {
+    volumeDown() {
         sendCommandByName(this, 'VolumeDown');
-    };
+    }
 
-    SessionPlayer.prototype.toggleFullscreen = function () {
+    toggleFullscreen() {
         sendCommandByName(this, 'ToggleFullscreen');
-    };
+    }
 
-    SessionPlayer.prototype.audioTracks = function () {
-        var state = this.lastPlayerData || {};
+    audioTracks() {
+        let state = this.lastPlayerData || {};
         state = state.NowPlayingItem || {};
-        var streams = state.MediaStreams || [];
+        const streams = state.MediaStreams || [];
         return streams.filter(function (s) {
             return s.Type === 'Audio';
         });
-    };
+    }
 
-    SessionPlayer.prototype.getAudioStreamIndex = function () {
-        var state = this.lastPlayerData || {};
+    getAudioStreamIndex() {
+        let state = this.lastPlayerData || {};
         state = state.PlayState || {};
         return state.AudioStreamIndex;
-    };
+    }
 
-    SessionPlayer.prototype.playTrailers = function (item) {
+    playTrailers(item) {
         sendCommandByName(this, 'PlayTrailers', {
             ItemId: item.Id
         });
-    };
+    }
 
-    SessionPlayer.prototype.setAudioStreamIndex = function (index) {
+    setAudioStreamIndex(index) {
         sendCommandByName(this, 'SetAudioStreamIndex', {
             Index: index
         });
-    };
+    }
 
-    SessionPlayer.prototype.subtitleTracks = function () {
-        var state = this.lastPlayerData || {};
+    subtitleTracks() {
+        let state = this.lastPlayerData || {};
         state = state.NowPlayingItem || {};
-        var streams = state.MediaStreams || [];
+        const streams = state.MediaStreams || [];
         return streams.filter(function (s) {
             return s.Type === 'Subtitle';
         });
-    };
+    }
 
-    SessionPlayer.prototype.getSubtitleStreamIndex = function () {
-        var state = this.lastPlayerData || {};
+    getSubtitleStreamIndex() {
+        let state = this.lastPlayerData || {};
         state = state.PlayState || {};
         return state.SubtitleStreamIndex;
-    };
+    }
 
-    SessionPlayer.prototype.setSubtitleStreamIndex = function (index) {
+    setSubtitleStreamIndex(index) {
         sendCommandByName(this, 'SetSubtitleStreamIndex', {
             Index: index
         });
-    };
-
-    SessionPlayer.prototype.getMaxStreamingBitrate = function () {
-
-    };
-
-    SessionPlayer.prototype.setMaxStreamingBitrate = function (options) {
-
-    };
-
-    SessionPlayer.prototype.isFullscreen = function () {
-
-    };
-
-    SessionPlayer.prototype.toggleFullscreen = function () {
-
-    };
-
-    SessionPlayer.prototype.getRepeatMode = function () {
+    }
 
-    };
+    setMaxStreamingBitrate(options) {
+    }
 
-    SessionPlayer.prototype.setRepeatMode = function (mode) {
+    setRepeatMode(mode) {
```

Comment:
Has `set`, but not `get` :confused: 
Is this method called for non-local player?
Or is it here just to be safe?

Reply:
I'm not too sure, I left it in just to be safe. If it isn't used, it could be removed.

Reply:
They (get/set) are used with same conditions.
https://github.com/jellyfin/jellyfin-web/blob/1339bb97f4161cddf2c9057a613c99f278e07bfc/src/components/playback/playbackmanager.js#L3552-L3567

`enableLocalPlaylistManagement` will return `false` for `sessionPlayer`.
https://github.com/jellyfin/jellyfin-web/blob/1339bb97f4161cddf2c9057a613c99f278e07bfc/src/components/playback/playbackmanager.js#L14-L24

So both are required.

Reply:
I have added `getRepeatMode` to sessionPlayer